src/main/java/com/quirko/gui/MenuPanel.java

```diff
@@ -0,0 +1,200 @@
+			if(e.getSource() == settings){
+				new SettingsMenuPanel();
+			}
+
+			else if(e.getSource() == load){
+				try {
+					load();
+				} catch (IOException e1) {
+					e1.printStackTrace();
+				}
+				GameController.getViewGuiController().refreshGameBackground(newGameMatrix);
+			}
+
+			else if(e.getSource() == save){
+				int[][] currentGame = SimpleBoard.getCurrentGameMatrix();
+				Score skor = SimpleBoard.getGameScore();
+				int score = skor.scoreProperty().get();
+				String str = "";
+				for (int i = 0; i < currentGame.length; i++) {
+					for (int j = 0; j < currentGame[i].length; j++) {
+						int index = currentGame[i][j];
+						str += String.valueOf(index);
+					}
+					str += "\n";
+				}
+				try {
+					save(Integer.toString(score), str, "game.txt");
+				} catch (IOException e1) {
+					e1.printStackTrace();
+				}
+			}
+
+			else if(e.getSource() == scoreBoard){
+				try {
+					howManyScore = 0;
+					loadScore();
+				} catch (IOException e1) {
+					e1.printStackTrace();
+				}
+				new ScoreBoardPanel();
+			}
+
+			else if(e.getSource() == exit){
+				System.exit(0);
+			}
+		}
+
+		@Override
+		public void keyPressed(KeyEvent e) {
+			if(e.getKeyCode() == KeyEvent.VK_ESCAPE){
+				MenuPanel.this.dispose();
+			}
+		}
+
+		@Override
+		public void keyTyped(KeyEvent e) {
+		}
+		@Override
+		public void keyReleased(KeyEvent e) {
+		}
+	}
+
+	// While saving, it takes the game screen matrix and the score method that writes to the game.txt file.
+	public void save(String score, String game, String name) throws IOException {
+		BufferedWriter bw = null;
+		bw = new BufferedWriter(new FileWriter("game.txt", false));
+		bw.write("game:\n");
+		bw.write(game);
+		bw.write("skor:\n");
+		bw.write(score);
+		bw.newLine();
+		bw.flush();
+		bw.close();
+	}
+
+	//Method that load the last saved game screen matrix and score ...
+	public void load() throws IOException {
+		String line = "";
+		BufferedReader br = new BufferedReader(new FileReader("game.txt"));
+		newGameMatrix = new int[SimpleBoard.getCurrentGameMatrix().length][SimpleBoard.getCurrentGameMatrix()[0].length];
+		int row = 0;
+		int skor = 0;
+		while ((line = br.readLine()) != null) {
+			if(line.equals("game:")){
+				while(true){
+					line = br.readLine();
+					if(line.equals("skor:")) break;
+
+					for(int col= 0; col < line.length(); col++){
+						newGameMatrix[row][col] = line.charAt(col) - '0';
+					}
+					row++;
+					SimpleBoard.setCurrentGameMatrix(newGameMatrix);
+				}
+			}
+
+			if(line.equals("skor:")){
+				line = br.readLine();
+				skor = Integer.parseInt(line);
+				SimpleBoard.getGameScore().reset();
+				SimpleBoard.getGameScore().add(skor);
+			}
+		}
+		br.close();
+	}
+
+	/*
+	The score "score.txt" whenever there is a gameover in the GuiController class -
+	If the user presses the scoreboard button, the scores will be
+	It is read from this file and printed on the screen.
+	*/ 
+	public void loadScore() throws IOException {
```

Comment:
Bu kısım başka bir issue da isteniyor. Bu kısmı boş bırak.

Sen yalnızca şablon olarak bırakacaksın burayı. O arkadaşın buraya daha sonra entegre olacak. Örneğin dosya ismi seçme, birden fazla save alma, automatic save almalar vs. gibi konular geçebilir o issueda. Bu kadar tek düze olması şart değil load/save kodunun.